### PR TITLE
bridge_ui: Chain Governor support

### PR DESCRIPTION
**Pull request by kev1n-peters**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1421)

### CODE REVIEW

File: bridge_ui/package-lock.json [hunk 1]
```diff
@@ -87,12 +87,12 @@
...
       "name": "@certusone/wormhole-sdk",
-      "version": "0.5.0",
+      "version": "0.6.1",
       "extraneous": true,
       "license": "Apache-2.0",
       "dependencies": {
-        "@certusone/wormhole-sdk-proto-web": "^0.0.1",
-        "@certusone/wormhole-sdk-wasm": "file:../js-wasm",
+        "@certusone/wormhole-sdk-proto-web": "^0.0.3",
+        "@certusone/wormhole-sdk-wasm": "^0.0.1",
```
Comment (panoel) [inline bridge_ui/package-lock.json:95]: Just out of curiosity, what is wormhole-sdk-wasm?

Comment (kev1n-peters) [reply bridge_ui/package-lock.json:95]: https://www.npmjs.com/package/@certusone/wormhole-sdk-wasm the wasm lib for invoking webassembly functions (mostly solana functions)

File: bridge_ui/src/components/Transfer/PendingVAAWarning.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  },
+}));
+
+const PendingVAAWarning = () => {
+  const classes = useStyles();
+  const sourceChain = useSelector(selectTransferSourceChain);
+  const chainName = CHAINS_BY_ID[sourceChain]?.name || "unknown";
+  const message = `The daily notional value limit for transfers on ${chainName} has been exceeded. As
+      a result, the VAA for this transfer is pending. If you have any questions,
+      please open a support ticket on `;
```
Comment (panoel) [inline bridge_ui/src/components/Transfer/PendingVAAWarning.tsx:20]: "open a support ticket on " ... Should there be a destination?  Or is that somewhere else?

Comment (kev1n-peters) [reply bridge_ui/src/components/Transfer/PendingVAAWarning.tsx:20]: the url is below in the <Link /> tag

File: bridge_ui/src/utils/getSignedVAAWithRetry.ts [hunk 3]
```diff
@@ -1,34 +1,42 @@
...
-        emitterChain,
-        emitterAddress,
-        sequence
-      );
-    } catch (e) {
-      if (retryAttempts !== undefined && attempts > retryAttempts) {
-        throw e;
-      }
+    const rpcHost = WORMHOLE_RPC_HOSTS[getNextRpcHost()];
+    const results = await Promise.allSettled([
```
Comment (panoel) [inline bridge_ui/src/utils/getSignedVAAWithRetry.ts:28]: Just out of curiosity, why is this an array?

**Review by panoel — APPROVED** [review]

File: bridge_ui/src/utils/getSignedVAAWithRetry.ts (hunk 3 above)
Comment (kev1n-peters) [reply bridge_ui/src/utils/getSignedVAAWithRetry.ts:28]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled#parameters

**Review by ckeun — APPROVED** [review]

**Pull request merged**